mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlime.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.3-rc"},
```

Comment:
Just as I suspected. The specs fail when attempting to do stick to `~> 1.2` with `allow_pre: false`